### PR TITLE
Just initializing the front end

### PR DESCRIPTION
**Pull request by lambrospap**

### CODE REVIEW

**Comment by lambrospap** [comment]
Initializing the front end to start working with it

File: i18n-ui/public/index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
```
Comment (igrigorov) [inline i18n-ui/public/index.html:27]: Change the title to Internationalization Administration

File: i18n-ui/src/App.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        <p>
+          Edit <code>src/App.js</code> and save to reload.
+        </p>
+        <a
+          className="App-link"
+          href="https://reactjs.org"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          Learn React
```
Comment (igrigorov) [inline i18n-ui/src/App.js:19]:
```suggestion
          Internationalization Administration
```

File: i18n-ui/src/backEnd.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+function fetch() {
+    return new Promise(resolve => setTimeout(() => resolve(42), 1000));
+}
+
+function fetchAPI(lng) {
+
+    return fetch("127.0.0.1:56727/l10n/locale/"+lng);
```
Comment (igrigorov) [inline i18n-ui/src/backEnd.js:9]: Is the port correct? On my machine it's 8080.

**Review by igrigorov — CHANGES_REQUESTED** [review]

File: i18n-ui/public/index.html (hunk 1 above)
Comment (igrigorov) [reply i18n-ui/public/index.html:27]: Changed.

File: i18n-ui/src/backEnd.js (hunk 3 above)
Comment (igrigorov) [reply i18n-ui/src/backEnd.js:9]: Changed.

**Review by igrigorov — APPROVED** [review]

**Pull request merged**